requirements/extra.in

```diff
@@ -0,0 +1,8 @@
+graphviz
+h5py
+matplotlib
```

Comment:
Did you mean `matplotlib-base` to avoid downloading the 50MB of Qt?
Also, it seems you are missing `ipympl`? I think we still need it for the docs (even though that might change in the future, as we discussed, we may want to remove the widgets around the plots in the docs).

Reply:
I don't think PyPI has `matplotlib-base`, appear to be a conda thing?

I'll add `ipympl`.

Reply:
I used to have to install `pyqt` explicitly when I used pip-based environments. So I guess it is not installed by default with matplotlib.